src/option.ts

```diff
@@ -128,11 +128,11 @@ export default class Option extends LitElement {
     this.#isActive = isActive;
 
     // Options are arbitrarily shown and hidden when Menu is opened and closed. So
-    // calling the below method in the `label` or `description` setters isn't sufficient
+    // calling the below method in the `label` or `description` setters isn't enough
```

Comment:
If you're a obsessive like I am, you can get a little cute to prevent the right edge of the comment block from becoming too jagged.